.github/workflows/Ci-raiwigets-python-typescript.yml

```diff
@@ -2,9 +2,9 @@ name: CI RAIWidgets Python Typescript
 
 on:
   push:
```

Comment:
Would this publish the npm packages for postBuild?

Reply:
This is just the build and test pipeline. No publishing included. You may be thinking about the CD pipeline, but I've excluded merged PRs to `postbuild` from triggering npm publish steps.